Bottom: df6b13d6ea97479a4e827c95b1198719983bbe35
Top:    f549536de064c41c97ab39d93c1c7e214477c2e5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:49:37 -0500

Refresh of playbook-extract-docker-hosts

---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index 67202894..cb652b51 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -1,6 +1,9 @@
 [bacula]
 daedalus.novatech-llc.com
 
+[daedalus]
+daedalus.novatech-llc.com
+
 [docker-hosts]
 daedalus.novatech-llc.com
 buildbot-worker.novatech-llc.com
